templates/statefulset-zabbix-proxy.yaml

```diff
@@ -83,7 +87,11 @@ spec:
 {{- toYaml .Values.zabbixproxy.resources | nindent 12 }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
+          {{- if .Values.zabbix_image_tag }}
+          image: "{{ .Values.zabbixproxy.image.repository }}:{{ .Values.zabbix_image_tag }}"
+          {{- else }}
           image: "{{ .Values.zabbixproxy.image.repository }}:{{ .Values.zabbixproxy.image.tag }}"
```

Comment:
Hi @sa-ChristianAnton!

With tag centralized, this line and the if can be removed, correct?

Reply:
commit 3f529c7